ui/components/app/flask/icons/verified-icon.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function VerifiedIcon({ className, color }) {
+  return (
+    <svg
+      className={className}
+      width="15"
+      height="14"
+      viewBox="0 0 15 14"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M12.8452 7C12.8452 8.48521 12.2482 9.90959 11.1855 10.9598C10.1228 12.01 8.68144 12.6 7.17855 12.6C5.67565 12.6 4.23432 12.01 3.17161 10.9598C2.1089 9.90959 1.51188 8.48521 1.51188 7C1.51188 5.51479 2.1089 4.09041 3.17161 3.0402C4.23432 1.99 5.67565 1.4 7.17855 1.4C7.71688 1.4 8.24105 1.477 8.73688 1.617L9.84897 0.518C9.0273 0.182 8.12772 0 7.17855 0C6.24835 0 5.32726 0.18106 4.46787 0.532843C3.60848 0.884626 2.82762 1.40024 2.16988 2.05025C0.841492 3.36301 0.0952148 5.14348 0.0952148 7C0.0952148 8.85652 0.841492 10.637 2.16988 11.9497C2.82762 12.5998 3.60848 13.1154 4.46787 13.4672C5.32726 13.8189 6.24835 14 7.17855 14C9.05717 14 10.8588 13.2625 12.1872 11.9497C13.5156 10.637 14.2619 8.85652 14.2619 7H12.8452ZM4.28146 5.656L3.28272 6.65L6.47022 9.8L13.5535 2.8L12.5548 1.806L6.47022 7.819L4.28146 5.656Z"
+        fill={color}
+      />
+    </svg>
+  );
+}
+
+VerifiedIcon.defaultProps = {
```

Comment:
Instead of using `defaultProps`, I think we can simply set a default value in the function signature:

```
export default function VerifiedIcon({ className, color ='#4CD964' }) {
```